drivers/storage/portworx/util/util.go

```diff
@@ -496,6 +500,17 @@ func GetClusterEnvVarValue(ctx context.Context, cluster *corev1.StorageCluster,
 	return ""
 }
 
+// GetPxProxyEnvVarValue returns the PX_HTTP(S)_PROXY environment variable value for a cluster.
+// Note: we only expect one proxy for the telemetry CCM container
+func GetPxProxyEnvVarValue(cluster *corev1.StorageCluster) string {
+	for _, env := range cluster.Spec.Env {
+		if env.Name == EnvKeyPortworxHTTPProxy || env.Name == EnvKeyPortworxHTTPSProxy {
```

Comment:
Did we confirm, if there is a precedence between https and http in case both are passed?

Reply:
who should I ask again? for the http_proxy file, it only expect one, so it's more like a question for portworx.